.env.local

```diff
@@ -30,6 +30,8 @@ TOT_MESSAGE_FETCH_WORKERS=5
 SHUFFLE_PINPAD_ON_PAYMENT=NO
 # Repository of app content
 CONTENT_REPO_URL='http://127.0.0.1:3000/static_contents'
+# Accepted card info page URL 
+PAYMENT_METHODS_PAGE_URL='https://io.italia.it/metodi-pagamento'
```

Comment:
You should add this url also in the others `.env` files
- .env.production
- .env.local
- .env.example

Reply:
we don't need to set this url as env variable.
It could be a const inside the code